Src/Plugins/Security/SEC101_050.SqlCredentialValidator.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Utilities;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Internal
```

Comment:
>Internal [](start = 71, length = 8)

remove internal

Reply:
Nice catch, done.

---
In reply to: [585941468](https://github.com/microsoft/sarif-pattern-matcher/pull/247#discussion_r585941468) [](ancestors = 585941468)